client-v2/src/shared/components/snapLink/SnapLink.tsx

```diff
@@ -80,16 +91,30 @@ const SnapLink = ({
             <CollectionItemMetaHeading>Snap link </CollectionItemMetaHeading>
           )}
           <CollectionItemHeading html>{headline}</CollectionItemHeading>
+          <SnapLinkURL>
+            snap link url: &nbsp;
```

Comment:
Probably just `url` is sufficient here -- the snap link is labeled just to the left of this component.